modules/ROOT/pages/administration-service.adoc

```diff
@@ -5,12 +5,12 @@ endif::[]
 
 :keywords: agent, mule, esb, servers, monitor, notifications, external systems, third party, get status, metrics
 
-image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-This document describes how to perform operations over the Runtime Manager Agent administration service API.
+This document describes how to perform operations over the Runtime Manager agent administration service API.
```

Comment:
What is the name of the API? Is it Runtime Manager Agent API? RMA Administration Service API? 
Typically here the APIs are named with the feature name, Runtime Manager API (and it has endpoints for agent and non-agent stuff).